### DISCUSSION TITLE
Benewake Lidar I2C Panic

### DISCUSSION THREAD

**Discussion by Some-Won-1** [iNavFlight/inav#7723] [Q&A]
Using 4.0.0 configurator with firmware iflight Twing F7 Dec 15 2021 14:11:09.  All UARTS in use so I put the TFMini-S Lidar in I2C mode using manufacture software.  It does exist on the same I2C bus with a compass just fine, but when BENEWAKE is selected and rebooted the FC beeps and flashes rapidly until Rangefinder is reverted back to NONE.  Sonar never goes blue.  Should this combination work on I2C or is this a Pull Request?

**Comment by Some-Won-1** [comment]
Some discussion with "Configurator" about this. #1209 ...Its a BIG assumption all this cool hardware is going to work with our software. Perhaps my dork intellect can spark a shazam.

**Comment by DzikuVx** [comment]
INAV does not support Benawake Lidar in I2C mode and as of now there are no plans to do so
